src/PDFPasswordForm.tsx

```diff
@@ -0,0 +1,164 @@
+import React, {useState, useRef, useMemo, type ChangeEvent, type FormEventHandler} from 'react';
+import PropTypes from 'prop-types';
+
+import {pdfPasswordFormStyles as styles} from './styles';
+
+type Props = {
+    isPasswordInvalid: boolean;
```

Comment:
```suggestion
    isPasswordInvalid?: boolean;
```

According to your propTypes.

Reply:
Done - 9ad9d67, 34f3717.